imap/mailbox.c

```diff
@@ -5536,9 +5536,7 @@ EXPORTED void mailbox_archive(struct mailbox *mailbox,
                     // didn't manage to store it, so remove the ARCHIVED flag
                     continue;
                 }
-                r = xunlink (srcname);
-                if (r < 0)
-                    syslog(LOG_ERR, "unlink(%s) failed: %m", srcname);
+                xunlink(srcname);
```

Comment:
Same here

Reply:
Same thinking as the earlier case -- we didn't care about any unlink failure except to log it, and we don't think ENOENT is worth logging because the end state is the state we were trying to reach by unlinking